### PR TITLE
GL-2055: Updated Build_indices container and Fixes on corresponding WDL

### PR DESCRIPTION
**Pull request by kevinpalis**
- The docker image builds successfully and is pushed to GCR
- Docker image follows our [guidelines](https://github.com/broadinstitute/warp/blob/develop/dockers/README.md)
- BuildIndices.wdl uses the new docker image
- Documentation has has been created for this image

WDL fixes:
- Changed the type of biotypes from String to File so it localizes properly
- Changed the genome_fa to use the reference’s value instead of a modified_genome_fa that didn’t exist (which STAR was looking for and was then failing)

Tested on Cromwell server https://cromwell.gotc-dev.broadinstitute.org
Succeeding run under the workflow ID: **41fa197d-1b47-4647-9d4f-6504b801c8f2**

### CODE REVIEW

**Comment by nikellepetrillo** [comment]
It looks like the Pipeline changelog validation and Pipeline release validation checks are failing. If you click into the Details, on the changelog validation test and then click console output, you will see in the log "BuildIndices.changelog.md has not been changed and needs to be updated" 

We have changelogs that sit next to each pipeline. Every time we update a pipeline or a task the pipeline uses, we need to:
1. update the changelog with a short description of the change you made 
2. update the pipeline_version in the pipeline wdl

File: dockers/skylab/build-indices/docker_versions.tsv
Comment (timaeusx) [inline dockers/skylab/build-indices/docker_versions.tsv:2]: For uniformity with the other images add `DOCKER_VERSION` to the topmost line and remove older versions not in use

Comment (kevinpalis) [reply dockers/skylab/build-indices/docker_versions.tsv:2]: Done.

**Review by timaeusx — APPROVED** [review]
Looks good!

**Comment by kevinpalis** [comment]
@nikellepetrillo - Looks like I need another reviewer before I can merge. Can you take a look? Thank you!

**Review by nikellepetrillo — APPROVED** [review]

**Pull request merged**
